dataland-frontend/src/components/forms/CreateEuTaxonomyForFinancials.vue

```diff
@@ -506,7 +510,29 @@ export default defineComponent({
               const receivedFormInputsModel = convertValuesFromDecimalsToPercentages(
                 companyAssociatedEuTaxonomyData as ObjectType
               );
+
+              const kpiSectionsModel = Object.keys(euTaxonomyKPIsModel.kpisFieldNameToFinancialServiceType)
+                .map( key => {
+                  const fieldName = euTaxonomyKPIsModel.kpisFieldNameToFinancialServiceType[key];
+                  const kpi = (receivedFormInputsModel.data as ObjectType).eligibilityKpis[fieldName];
```

Comment:
`kpi` is wrong naming. Basically a kpi corresponds to a single data point. `kpis` would fit. Better `eligibilityKpis`.

Reply:
Changed it to `eligibilityKpi`, but yes, we're getting a single data point here.

Reply:
We are not looking on a single data point but on the collection of all eligibility related datapoints of one of the financial service types (`fieldName`).

Reply:
Done.